Wikipedia/Code/NotificationsCenterFilterView.swift

```diff
@@ -65,6 +66,7 @@ struct NotificationsCenterFilterItemView: View {
                 }
             }
         }
+        .padding(.horizontal, horizontalSizeClass == .regular ? 16 : 0)
```

Comment:
I wonder if SwiftUI or the accessibility APIs give us a better constant for this that is more dependent on their dynamic type sizing. This is working fine for me though, so I would say looking into this is optional.

Edit: Maybe [this API](https://developer.apple.com/documentation/uikit/uifontmetrics/2877388-scaledvalue) would help.

Reply:
Done, thanks!